xml/System/GCMemoryInfo.xml

```diff
@@ -234,7 +243,8 @@ The memory after `OBJ_D` is not considered part of the `FragmentedBytes` but wil
         <ReturnType>System.Int64</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
+        <summary>The index of this GC. GC indices start with 1 and get increased at the beginning of a GC.
+            Since the info is updated at the end of a GC, this means you can get the info for a BGC with a smaller index than a foreground GC finished earlier.</summary>
```

Comment:
```suggestion
            Since the information is updated at the end of a GC, this means you can get the information for a BGC with a smaller index than a foreground GC finished earlier.</summary>
```